boards/arm/contextualelectronics_abc/board.cmake

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set (OPENOCD_NRF5_SUBFAMILY "nrf52")
+board_runner_args(nrfjprog "--nrf-family=NRF52")
```

Comment:
This line is not needed anymore since 6628a16e4d7aa46e57fd2cc8796592983bcc6516; please remove.

I've posted a separate PR (https://github.com/zephyrproject-rtos/zephyr/pull/31121) eliminating it from elsewhere in the tree.